pages/verify/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -6,14 +6,16 @@ import { connect as starknetConnect, disconnect } from "starknetkit";
 
 import Logo from "../../../../components/Logo";
 import SocialLinks from "../../../../components/SocialLinks";
-import chainAliasByNetwork from "../../../../configs/chainAliasByNetwork.json";
+// import chainAliasByNetwork from "../../../../configs/chainAliasByNetwork.json";
 import { DiscordMemberRepository, setupDb } from "../../../../db";
 import { getDiscordServerInfo } from "../../../../discord/utils";
 import { NetworkName } from "../../../../types/starknet";
 import messageToSign from "../../../../utils/starknet/message";
 import WatchTowerLogger from "../../../../watchTower";
 import DiscordServerInfo from "../../../../components/verification/DiscordServerInfo";
 import WalletInfo from "../../../../components/verification/WalletInfo";
+import rawChainAlias from "../../../../configs/chainAliasByNetwork.json";
+const chainAliasByNetwork = rawChainAlias as Record<NetworkName, string[]>;
```

Comment:
Why not just importing it as we did before ?
Your issue is not about this page so I don't think those changes are needed